pkg/syncer/metadatasyncer.go

```diff
@@ -217,6 +217,21 @@ func InitMetadataSyncer(ctx context.Context, clusterFlavor cnstypes.CnsClusterFl
 						time.Sleep(5 * time.Second)
 					}
 				}
+				if metadataSyncer.clusterFlavor == cnstypes.CnsClusterFlavorWorkload {
```

Comment:
Since we're only watching onthe ca file path if cluster flavour is et to workload (line251), is this check needed?

Reply:
I refactored the current ReloadConfiguration function to be re-used for ca file rotation which internally has the cluster flavor check. PTAL